documentation/api/processes-list.md

```diff
@@ -10,14 +10,18 @@ GET /processes HTTP/1.1
 
 > **NOTE:** Process information (IDs, names, environment, etc) may change between invocations of these APIs. Processes may start or stop between API invocations, causing this information to change.
 
+## Host Address
+
+The default host address for these routes is `https://localhost:52323`. This route is only available on the addresses configured via the `--urls` command line parameter and the `DOTNETMONITOR_URLS` environment variable.
```

Comment:
nit: I'm not sure we need to call this out in every doc.

Reply:
Some doc needs to call out which routes are available on which addresses. This seemed like a natural place.

This is alos not a boilerplate statement; this statement is in contrast to the /metrics.md doc, which calls out that it is available on all addresses.

Reply:
That's a good point, that restriction is not very obvious.